packages/ui/components/elements/IngredientSearchInput/IngredientSearchInput.tsx

```diff
@@ -6,22 +6,22 @@ import { uuidv4 } from '../../utils/uuid'
 
 type IngredientSearchInputProps = {
   id: string
-  imgSrc: string
-  imgAlt: string
+  imgSrc?: string
+  imgAlt?: string
 }
 
 const SearchIcon = () => (
   <div className=''>
     <Icon icon={search} size={24} />
   </div>
 )
-const IngredientSearchInput = () => {
-  const id = `input-search-field-${uuidv4()}`
+const IngredientSearchInput = ({ id, imgSrc, imgAlt }: IngredientSearchInputProps) => {
+  // const id = `input-search-field-${uuidv4()}`
```

Comment:
We can remove the commented codes as it shows our code cleaner.  Besides, we have git to see the previous versions.